Sources/LiveViewNative/Property Wrappers/LiveBinding.swift

```diff
@@ -0,0 +1,215 @@
+//
+//  LiveBinding.swift
+//  LiveViewNative
+//
+//  Created by Shadowfacts on 1/25/23.
+//
+
+import SwiftUI
+import LiveViewNativeCore
+import Combine
+
+/// Live bindings provide a mechanism for sharing state between the server and client in a way that can be updated by either.
+///
+/// Updates to the value from the client-side will automatically send an event to the server to update its stored value.
+/// Likewise, whenever the value is updated on the server, an event will be sent to the client which will trigger the `@LiveBinding` to update and the SwiftUI view that contains it (as well as any that use the ``projectedValue`` binding) to re-render.
+///
+/// ### Binding Names
+/// The name of a live binding is not defined directly by the client.
+/// Instead, it always controlled by the backend, to prevent the name getting out-of-sync, especially when multiple client versions may be in use.
```

Comment:
```suggestion
/// Instead, it is always controlled by the backend, to prevent the name getting out-of-sync, especially when multiple client versions may be in use.
```